package-lock.json

```diff
@@ -1,3958 +1,8 @@
 {
   "name": "sway-vscode-plugin",
   "version": "0.2.8",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
I guess the difference is the npm version. npm v7 creates `lockfileVersion: 2`. I think we should probably switch to yarn because it's faster and better 😄